BUILDING.txt

```diff
@@ -14,10 +14,17 @@ http://maven.apache.org/run-maven/index.html
 
 (2) Executing test cases
 
-Execute the following command in order to compile and test the components
+Execute the following command in order to compile and test the components:
 
 mvn test
 
+To avoid intermittent test failures (i.e., flaky tests), it is recommended to have a machine or virtual machine with the following specifications:
+
+Minimum of 2 CPUs.
+Minimum of 2GB of RAM.
+
+By meeting these requirements, you can mitigate potential flaky tests and ensure a smooth testing experience.
+
```

Comment:
@Agorguy Can we rephrase this sentence to make it sound a bit less frivolous and avoid the term "flaky"

```
These are the minimal requirements for stable test execution. In more resource constrained environments some tests may fail intermittently
```
or some such? 

Do tests _really_ need 2 CPUs?

Reply:
I updated that sentence, your suggestion was great.

Yes, when running with less than 2CPU the chances of flakies appearing are greater.

Reply:
> Yes, when running with less than 2CPU the chances of flakies appearing are greater.

@Agorguy Bizarre. But it is good to know. Now, the real thing is to find out why.